src/Language/PureScript/Sugar/Operators.hs

```diff
@@ -57,9 +40,24 @@ import qualified Language.PureScript.Constants as C
 rebracket :: (Applicative m, MonadError MultipleErrors m) => [ExternsFile] -> [Module] -> m [Module]
 rebracket externs ms = do
   let fixities = concatMap externsFixities externs ++ concatMap collectFixities ms
-  ensureNoDuplicates $ map (\(i, pos, _) -> (i, pos)) fixities
-  let opTable = customOperatorTable $ map (\(i, _, f) -> (i, f)) fixities
-  traverse (rebracketModule opTable) ms
+  ensureNoDuplicates $ map (\(i, pos, _, _) -> (i, pos)) fixities
+  let opTable = customOperatorTable $ map (\(i, _, f, _) -> (i, f)) fixities
+  ms' <- traverse (rebracketModule opTable) ms
+  let aliased = M.fromList (mapMaybe makeLookupEntry fixities)
+  return $ renameAliasedOperators aliased `map` ms'
+
+  where
+
+  makeLookupEntry :: (Qualified Ident, SourceSpan, Fixity, Maybe Ident) -> Maybe (Qualified Ident, Qualified Ident)
+  makeLookupEntry (qname@(Qualified qual _), _, _, Just alias) = Just (qname, Qualified qual alias)
+  makeLookupEntry _ = Nothing
+
+  renameAliasedOperators :: M.Map (Qualified Ident) (Qualified Ident) -> Module -> Module
+  renameAliasedOperators aliased (Module ss coms mn ds exts) = Module ss coms mn (map f' ds) exts
+    where
+    (f', _, _) = everywhereOnValues id go id
+    go (Var name) = Var $ fromMaybe name (name `M.lookup` aliased)
```

Comment:
No, because this matches all `Var`s, the map only has the aliased operators in it, so if the lookup fails it just means it's not an operator, or one that was defined the old way.